knora/dsplib/utils/xml_upload.py

```diff
@@ -355,7 +356,7 @@ def _upload_resources(
     """
 
     # If there are multimedia files: calculate their total size
-    bitstream_all_sizes_mb = [os.path.getsize(res.bitstream.value) / 1000000 for res in resources if res.bitstream]
```

Comment:
the old code was broken if a different imgdir was chosen. So it is necessary to take account of this